### DISCUSSION TITLE
Testing Idena node: release candidate 0.30.1-rc0

### DISCUSSION THREAD

**Discussion by midenaio** [idena-network/idena-go#1028] [General]
We invite you to test the release candidate version of the Idena node: `0.30.1-rc0`
Please [download](https://github.com/idena-network/idena-go/releases/tag/v0.30.1-rc0) the node manually if you want to join the testing.

Changes:
- IPFS version upgrade
- Truncating blockchain history (the last hard fork update genesis block is used for sync)
- Compacting blockchain data when the node is starting
- Bug fixes

We plan to release these changes soon unless negative feedback is received.

**Comment by idena-Egyptian-Pharoh** [comment]
Ready to use the real version 
I think it's a good time to truncate the Blockchain
Because it's about 1.7 gb now and costs alot of bandwidth as i hold the fast sync idenachain.db.zip

**Comment by rioda-org** [comment]
So far so good. After syncing, node settled at 360MB of ram.

**Comment by Zen-44** [comment]
Seems to be working fine.

**Comment by rioda-org** [comment]
Well, after released version and few days, node seems to take up much more ram memory.
3GB in my case.

**Reply by hlolve** [reply]
Yes, there is an increase in memory use over time, like a memory leak. For test I run 3 instances in same vps, and the memory consumed by each can be different about 3x, running since same time and not stopping them.

**Reply by hlolve** [reply]
Also seems that in multi node mode affect more than in single node.
